packages/perseus/src/renderer.tsx

```diff
@@ -603,7 +607,6 @@ class Renderer extends React.Component<Props, State> {
 
         return {
             ...widgetProps,
-            ref: id,
```

Comment:
This ref prop was never used, and it is a huge potential bug. Now that we use proper refs in the `WidgetContainer`, we can safely remove this.